### DISCUSSION TITLE
how to run commands if renamed on server

### DISCUSSION THREAD

**Discussion by tessus** [phpredis/phpredis#2596] [Q&A] [answered]
How can I run `flushDb` or any other command that has been renamed on the server?

**Comment by yatsukhnenko** [comment, marked as answer]
You can use [rawCommand](https://github.com/phpredis/phpredis/blob/develop/README.md#rawcommand)

**Reply by tessus** [reply]
Perfect thanks. `rename-command` is deprecated anyway and I hope it will be removed soon.
Unfortunately until then one might have to support this and with `rawCommand` it is possible.
